### PR TITLE
add MIT LICENSE

### PR DESCRIPTION
**Pull request by NerOcrO**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 NerOcro <nerocro@gmail.com>
```
Comment (maxlath) [inline LICENSE:3]: ^^

**Review by maxlath — CHANGES_REQUESTED** [review]

**Comment by NerOcrO** [comment]
Too fast...
What do you want?

**Comment by NerOcrO** [comment]
Hello,
Can you respond me please?

**Comment by maxlath** [comment]
* no we won't give you full copyright on this repo (see review above which didn't get an answer from you)
* if a license should be added, it will rather be AGPL-3, to be consistent with other inventaire projects

**Comment by NerOcrO** [comment]
Putain sérieux, à quel moment j'ai voulu avoir les droits sur ton dépôts ???
C'était juste une typo et je t'ai demandé ce que tu voulais à la place et tu ne m'as jamais répondu...
J'suis même venu sur ton chan IRC à l'époque et personne ne me répondait...
Si tu voulais un autre type de licence, il fallait me le dire au lieu de mettre un smiley qui ne veut strictement rien dire...
Ca me fait halluciné de voir des réponses comme ça alors que l'on essaye d'aider !
Ne mets pas ton projet en open source si tu n'es pas ouvert aux modifications.

**Comment by jum-s** [comment]
With a smiley, @maxlath was refering to the insertion [this line](https://github.com/inventaire/stack/blob/d17fcb35e37b9244c354ae4f6de401c618a73a19/LICENSE#L3) which contains `Copyright (c) 2018 NerOcro <nerocro@gmail.com>` which requested changes in order to be merged, since no copyright is authorized among inventaire's repos. 
@maxlath comment could have been longer and i understand that it could be frustrating to not have more explanations, but may i remind that maintaining a foss project like inventaire.io is a quite some work, which means that the team dont necessarily have time to post long answers (like this one) and rather keep it efficient and short in order to have time to improve the code base. That also explain why we do not chitchat on inventaire IRC, the channel is more used to keep updated of occasionally news (if you kept going on the channel, you would have noticed...) 
In my opinion, your last comment is aggressive (cf. "Putain sérieux") and besides the fact that this comment will stay 'forever' online, in a professional environment this tone is quite conterproductive and undermine possibilities of moving on. By the way, most of your PRs were merged on this repo, which means inventaires project is now better and this is thanks to you.

**Pull request closed without merging**